examples/animation/main.js

```diff
@@ -172,6 +172,7 @@
                 config.currentAnim = config.anim || firstAnimation;
                 if ( config.currentAnim ) {
                     controller.play = function () {
+                        animationManager.stopAllAnimation();
```

Comment:
stopAllAnimation, just reset animation playing. no dirty or recomputing is involve here